content/node/trustlessStaking.md

```diff
@@ -0,0 +1,120 @@
+See [stake-weighted-servicer-rewards](/learn/economics/nodes/#stake-weighted-servicer-rewards) for more info
+
+
+
+**How non-custodial looks like on Poktscan**
+
+https://poktscan.com/node/bab0bc034cbf4ae82a5069b677e8c2f39098e361
+![](/images/poktscan-noncustodial.png)
+
+Note the stake type “Non-Custodial”. Also note that the address of the node is different from the output address. In the example above:
+
+**Operator Address:**  bab0bc034cbf4ae82a5069b677e8c2f39098e361  
+**Operator Address Public Key:** d7a3f683341a5fd82adf4f4411544527ba15726d9558c5b83b9a2c6946b4df44  
+**Output Address:** 3574ca25de8744aa167cff234b69b8a4eef97711
+
+Notice there is a balance of 36.57 present. Node requires some pokt to pay to claim rewards and also any staking operations(edit domain/chain) node operator will do. Either you or the node operator can keep the operator account topped up periodically.
+
+**IMPORTANT:**  Any balance in the operator address is accessible by the node operator, do not send any amount other than what you intend to be used as fees or better let your node operator handle all fees.
+
+***Checklist***
+- [ ] My output address is _________________. I want my staked funds(when unstaked) and rewards here. I have backed it up safely and I will not share the privatekeys/keyfile of this address ever.
+
+##  Information to get from your Node Operator:
+
+***Checklist***
+- [ ] Operator Address:  __________________________________
+- [ ] Operator Public Key:  __________________________________ 
+- [ ] Chains your Nodes Operator will support:  __________________________________
+- [ ] Domain:  __________________________________
+
+## Preparing the Output Address
+
+1. Login to your  output address using the pocket official wallet below. Note down the password used to create the wallet. https://wallet.pokt.network/
+2. **IMPORTANT:** Backup the keyfile.json using “export keyfile” and the password. Lose this and you will lose your funds. Then send 15011 tokens to this address. 15010 will be staked, leaving one for transaction fees.(Each transaction costs 0.01 pokt at the time this doc is written)
+ 
+![](/images/wallet.png)
+
+## Staking Non-Custodial
+
+1. Login from output address
+2. Go to the staking Tab: https://wallet.pokt.network/staking
+3. You should already have the operator public key, output address,chains and domain from the checklists. 
+
+{{% notice style="warning" %}}
+**1. Wrong output address will result in permanent loss of funds**
+
+**2. Wrong operator public key may lock your funds for 21 days(you have to unstake to get your funds back)**
+{{% /notice %}}
+
+![](/images/wallet-staking.png)
+
+4. Hit the Stake Node Button, verify your stake on poktscan in 15-20 minutes.
+
+***Checklist***
+- [ ] I verified that it is staked non-custodial on poktscan at the url https://poktscan.com/node/\<myoperatoraddress\> with the correct output address.
```

Comment:
> it is staked non-custodial on poktscan at the url https://poktscan.com/node/\<myoperatoraddress\> with the correct output address.

the node is staked non-custodial on [POKTScan](https://poktscan.com/explore?tab=nodes) using my output address to search for the account.

Reply:
beginners can't figure out poktscan initially. I wanted to give them a direct url where they can check their node.

Reply:
I changed to this I verified that the node is staked non-custodial on poktscan at the url https://poktscan.com/node/<myoperatoraddress> and it is showing the correct output address.